src/daemon/SeatManager.cpp

```diff
@@ -152,6 +157,23 @@ namespace SDDM {
         m_seats.value(name)->createDisplay();
     }
 
+    // this is a bit hacky, but linux DRM drivers
+    // won't initially be available so there is a race
+    // between determing if a efifb/etc graphical object
+    // is the only graphics on the machine, or a DRM driver
+    // will take over the display. So we will hang out for a few
+    // seconds and if none of the seats are declared cangraphical
+    // its possible the only graphics on the machine don't have
+    // a drm driver.
+    void SeatManager::checkSeat(void) {
+        if (m_seats.isEmpty()) {
+	     //if (QFileInfo::exists(QStringLiteral("/dev/fb0"))) {
```

Comment:
We can probably remove the commented if?

Reply:
Fixed.

Reply:
I can still see them :eye: